pkg/api/controller.go

```
@@ -1863,15 +1868,10 @@ func (c *Controller) GetConfig(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
-	// TODO (gilo): is StorageID relevant here?
-	storageCfg, err := c.getStorageConfig("")
-	if c.handleAPIError(r.Context(), w, r, err) {
-		return
-	}
-	// TODO (niro): Needs to be populated
-	storageListCfg := apigen.StorageConfigList{}
+	storageCfg, _ := c.getStorageConfig(config.SingleBlockstoreID)
```

Is it ok that we swallowed the error here?